frontend/src/bookings/calendar.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div>
+    <div class="title inline-title">{{ title }}</div>
+    <keep-alive>
+      <full-calendar ref="qcalendar"
+                     key="qcalendar"
+                     class="calendar-margins"
+                     @view-render="viewRender"
+                     @event-render="eventRender"
+                     :events="events"
+                     :config="configuration"></full-calendar>
+    </keep-alive>
+  </div>
+
+</template>
+
+<script>
+  import { mapState, mapMutations, mapActions } from 'vuex'
+  import 'fullcalendar/dist/fullcalendar.css'
+  import 'fullcalendar-scheduler'
+
+  export default {
+    name: 'Calendar',
+    created() {
+      this.getExams()
+      this.getBookings()
+      this.getRooms()
+    },
+    mounted() {
+      this.$root.$on('next', () => { this.next() })
+      this.$root.$on('prev', () => { this.prev() })
+      this.$root.$on('today', () => { this.today() })
+      this.$root.$on('month', () => { this.month() })
+      this.$root.$on('listWeek', () => { this.listWeek() })
+      this.$root.$on('listDay', () => { this.listDay() })
+    },
+    data() {
+      return {
+        config: {
+          schedulerLicenseKey: 'GPL-My-Project-Is-Open-Source',
+          showNonCurrentDates: false,
+          fixedWeekCount: false,
+          navLinks: true,
+          defaultView: 'listWeek',
+          views: {
+            listDay: {
+              allDaySlot: false,
+            },
+            listWeek: {
+              allDaySlot: false,
+            },
+          },
+          height() {
+            return window.innerHeight - 230
+          },
+          weekends: false,
+          maxTime: '18:00:00',
```

Comment:
Can you confirm with John what times this should display? Do you also know how this plays with timezones? E.g. if someone from PST edits an office in MST, or vice versa? We might need an extra hour of buffer (MST offices start at 7 AM PST)